types/src/version.rs

```diff
@@ -41,6 +41,8 @@ pub enum NetworkVersion {
     V13,
     /// chocolate (specs-actors v6.0.0)
     V14,
+    // TBD
```

Comment:
I believe the name is OhSnap?

Reply:
That's korrekt.

Reply:
See duplicated version [here](https://github.com/filecoin-project/ref-fvm/blob/6f3db40ad7108e27146d0b9efdf7c64107507839/shared/src/version/mod.rs)
Related issue #1448.